frontend/src/contracts.ts

```diff
@@ -25,10 +25,72 @@ export function useDAOv1(): ComputedRef<DAOv1> {
   });
 }
 
+export function useUnwrappedDAOv1(): ComputedRef<DAOv1> {
```

Comment:
Could you move `useUnwrappedDAO` and `useUnwrappedPollACLv1` to a separate PR please? I would need that for my GSN voting as well and let's merge it separately ;)